### PR TITLE
LPD-30855 Refactor poshi test 

### PR DESCRIPTION
**Pull request by monaco-fabio**
https://liferay.atlassian.net/browse/LPD-30855

### CODE REVIEW

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:playwright:nocomile

**Comment by monaco-fabio** [comment]
ci:test:playwright:nocompile

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:playwright:nocompile

**Comment by andrea-ale-sbarra** [comment]
The test didn't pass on CI, can you please rebase and check @monaco-fabio.
Thank you!

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:playwright:nocompile

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:playwright:nocompile

File: modules/test/playwright/tests/commerce/utils/date.ts [hunk 1]
```diff
@@ -26,3 +26,17 @@ export function getDateCustomFormat(
 
 	return date?.toLocaleDateString('en-US', format);
 }
+
+export function getUTCHourAndDateFormat(date: string) {
+	return new Date(date)
+		.toLocaleDateString('en-US', {
```
Comment (andrea-ale-sbarra) [inline modules/test/playwright/tests/commerce/utils/date.ts:32]: shouldn't you take the same locale of the CI?

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:stop

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:playwright:nocompile

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:playwright:nocompile

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.
Thank you!

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Pull request closed without merging**